superset/assets/javascripts/dashboard/reducers/allSlices.js

```diff
@@ -0,0 +1,74 @@
+import * as actions from '../actions/allSlices';
```

Comment:
please import individual actions, it is
1) less error prone and
2) can be linted in the future, `*` breaks linting.